sql/2019/08_Security/08_08.sql

```diff
@@ -0,0 +1,14 @@
+#standardSQL
+# 08_08 Modern cipher suites 
+#             | TLS_AES_128_GCM_SHA256       | {0x13,0x01} |
```

Comment:
So, these are just the TLS 1.3 ciphers which are indeed modern :D but we probably want to include modern TLS 1.2 ciphers as well (the various AES GCMs, and ChaCha20-Poly1305s).

I'll look into cimpiling a list with the additional ciphers.

Reply:
Sorry for the delay. Here is the list of ciphers:

### TLS 1.3 ciphers:

0x1301 	TLS_AES_128_GCM_SHA256
0x1302 	TLS_AES_256_GCM_SHA384
0x1303 	TLS_CHACHA20_POLY1305_SHA256
0x1304 	TLS_AES_128_CCM_SHA256

### TLS 1.2 ciphers:

0xC02B 	TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
0xC02C 	TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
0xC02F 	TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
0xC030 	TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
0xCCA8 	TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
0xCCA9 	TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256
0xCCAC 	TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305_SHA256
0xD001 	TLS_ECDHE_PSK_WITH_AES_128_GCM_SHA256
0xD002 	TLS_ECDHE_PSK_WITH_AES_256_GCM_SHA384
0xD005 	TLS_ECDHE_PSK_WITH_AES_128_CCM_SHA256

This is basically the list of ciphers that have "Y" in the "Recommended" columns at https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4

From that list I removed the DHE ciphers since they are not used in practice anyway (though I can add them back if people disagree).